docs/ref/django-admin.txt

```diff
@@ -91,7 +93,9 @@ Uses the :doc:`system check framework </ref/checks>` to inspect the entire
 Django project for common problems.
 
 By default, all apps will be checked. You can check a subset of apps by
-providing a list of app labels as arguments::
+providing a list of app labels as arguments:
+
+.. console::
```

Comment:
if it is equivalent to `code-block::shell` perhaps it would be better to use the same mode everywhere.

Reply:
It's not. It gives a special windows/*nixy tabbed interface. Maybe there's a case for making some extra cases use it, but I'd like to address that separately, as it needs thinking about. (e.g. The GIS docs are very OS specific, and so on. )

Reply:
I didn't know about this special behavior of `console`.
It's perfectly fine to leave it at that.